packages/trimerge-sync-indexed-db/src/trimerge-indexed-db.ts

```diff
@@ -186,7 +186,12 @@ class IndexedDbBackend<
     metadata: CommitMetadata | undefined,
     remoteSyncId: string | undefined,
   ): commit is TrimergeSyncDbCommit<CommitMetadata, Delta> {
-    if (commit && !commit.remoteSyncId && remoteSyncId) {
+    // In the world where servers decide when to send merges,
+    // it's conceivable that a commit could be "acked" twice
```

Comment:
Depends what you mean by acked.  Multiple posts with a commit will 200, but a given commit will only ever come back once in post/get responses -- which could be significantly (or infinitely) delayed from when it was submitted in a post.